src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -30,24 +29,23 @@ public static class TextCatalog
         /// </example>
         public static TextFeaturizingEstimator FeaturizeText(this TransformsCatalog.TextTransforms catalog,
             string outputColumnName,
-            string inputColumnName = null,
-            Action<TextFeaturizingEstimator.Settings> advancedSettings = null)
+            string inputColumnName = null)
             => new TextFeaturizingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(),
-                outputColumnName, inputColumnName, advancedSettings);
+                outputColumnName, inputColumnName);
 
         /// <summary>
         /// Transform several text columns into featurized float array that represents counts of ngrams and char-grams.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.</param>
         /// <param name="inputColumnNames">Name of the columns to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="advancedSettings">Advanced transform settings</param>
+        /// <param name="options">Advanced transform settings.</param>
```

Comment:
> /// <param name="options">Advanced transform settings.</param> [](start = 7, length = 63)

@sfilipi any chance we have a consensus regarding what to put here?
I can see ` /// <param name="options">Advanced arguments to the algorithm.</param>`
`/// <param name="options">Algorithm advanced options.</param>`
`/// <param name="options">Algorithm advanced settings.</param>`
`/// <param name="options">Advanced options to the algorithm.</param>`

Reply:
Issue #2154 to fix this across codebase  (once we finalize what to put in here) 

---
In reply to: [254018114](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254018114) [](ancestors = 254018114)

Reply:
Since the object's name is now Options, let;s go with:

/// <param name="options">Advanced options to the algorithm.</param>

if you can search/replace in one of the PR that would be awesome!

---
In reply to: [254018114](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254018114) [](ancestors = 254018114)

Reply:
will do. for now i have fixed this one

---
In reply to: [254356441](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254356441) [](ancestors = 254356441,254018114)